visualizer/options-menu.css

```diff
@@ -90,8 +128,54 @@
 #options-menu .options input {
   margin: 0 8px 0 0;
   vertical-align: middle;
+  display: none;
 }
 
+#options-menu .options .icon-wrapper {
+  align-items: center;
+  border-radius: 3px;
+  display: flex;
+  font-size: 1.4em;
+  margin-right: 0.5em;
+  flex-shrink: 0;
+  justify-content: center;
+  color: var(--checkbox-border-color);
+}
+#options-menu .options .icon-wrapper svg{
+  display: none;
+}
+#options-menu .options .icon-wrapper .unchecked{
+  display: initial;
+}
+
+#options-menu .options :checked ~ .icon-wrapper .unchecked{
+  display: none;
+}
+
+#options-menu .options :checked ~ .icon-wrapper{
+  color: var(--max-contrast);
+}
+
+#options-menu .options :checked ~ .icon-wrapper .checked{
```

Comment:
It shouldn't show two icons because a checkbox can't be checked and indeterminate at the same time, right?
Is it showing two icons to you? It looks fine to me

Reply:
I get this on Firefox:

![image](https://user-images.githubusercontent.com/1006268/48123295-d824a300-e279-11e8-806e-f2308b6268cd.png)

indeterminate state is a separate thing from checked/unchecked state, so FF probably reads both the `input.checked` and `input.indeterminate` properties. maybe other browsers are more sensible and check `input.indeterminate`, then decide not to apply `:checked` pseudo-classes

Reply:
You are right, some browsers are pickier than others 😉